MdeModulePkg/Universal/PCD/Pei/Pcd.c

```diff
@@ -316,6 +324,14 @@ EndOfPeiSignalPpiNotifyCallback (
   if (PcdDb != NULL) {
     Length   = PeiPcdDb->LengthForAllSkus;
     Database = BuildGuidHob (&gPcdDataBaseHobGuid, Length);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (Database == NULL) {
+      DEBUG ((DEBUG_ERROR, "Failed build PCD guid hob.\n"));

```

Comment:
should the debug messages include the **__FUNCTION__** maco?

Reply:
If you add it, remember to use `__func__` for new code.

Reply:
Updated.